src/Exception/BaseException.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\SnowflakeDbAdapter\Exception;
+
+class BaseException extends \Exception implements SnowflakeDbAdapterException
```

Comment:
Já mam `Base` prefixed classy spojený spíš s abstrakcí. Pokud se přejmenuje ten Interface, tak by se to mohlo jmenovat  `SnowflakeDbAdapterException`